js/scores.js

```diff
@@ -20,6 +22,46 @@ function makeTable(){
   }
 }
 
+var createSQL = function(callback) {
+  webDB.execute(
+    [
+      {
+        'sql': 'CREATE TABLE IF NOT EXISTS highscores ( id INT, n TEXT, s INT, PRIMARY KEY (id));',
```

Comment:
Your `id` column is not set up correctly for WebSQL; the id column remains blank for all inserted records. Also, try to use more semantic names for your database columns.